kcidev/libs/maestro_common.py

```diff
@@ -53,10 +54,10 @@ def maestro_get_node(url, nodeid):
         response.raise_for_status()
     except requests.exceptions.HTTPError as ex:
         kci_err(ex.response.json().get("detail"))
-        click.Abort()
+        sys.exit(errno.ENOENT)
```

Comment:
why use errno.ENOENT? I thought was referred only to local directory not about network URL paths
ENOENT means "No such file and directory", and is for path operations.
https://github.com/torvalds/linux/blob/5c8c229261f14159b54b9a32f12e5fa89d88b905/include/uapi/asm-generic/errno-base.h#L6